beta-prompts/beta_prompts/improvement_engine.py

```
@@ -0,0 +1,610 @@
+
+        return results
+
+    def save_optimization_results(
+        self,
+        results: list[OptimizationResult],
+        filename: str = "prompt_optimization_results.json",
+    ):
+        """Save optimization results to file"""
+
+        data = {
+            "optimization_date": datetime.now().isoformat(),
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)